### ISSUE TITLE
Terminal.IsConsolePresent exception when Console is not supported

### ISSUE THREAD

**Issue by AbeniMatteo** [unosquare/swan#191]
**Describe the bug**
I'm exploring the possibility to use EmbedIO on **Android** using Xamarin.
`NotSupportedException` is thrown initializing Embedio's `WebServer` because `Console` is not available on this platform.

```
>   at System.Console.get_WindowHeight () [0x00000] in <3e43eda906c6495eb12edfbee9e53c18>:0 
>   at Swan.Terminal.get_IsConsolePresent () [0x00020] in <70e2ad446e2f4b218662d16bd08415b9>:0 
```

There is `Terminal.Settings.OverrideIsConsolePresent` flag but it can only be used to force-enable the presence, after that the same exception is thrown at the next `Console` use.

**Expected behavior**
Exception catched internally and `IsConsolePresent` set to false or being able to disable manually the detection setting `Terminal.Settings.OverrideIsConsolePresent` to false.

**Platform**
 - Device: Nexus 5 emulator
 - OS: Android 9.0 - API 28

**Comment by AbeniMatteo** [comment]
My bad, the exception is caught correctly, Visual Studio stops the execution incorrectly reporting it as a first chance user-code exception.

The discussion about `OverrideIsConsolePresent` remains valid.

**Comment by geoperez** [comment]
I think I need to remove `OverrideIsConsolePresent` property. This is a legacy property of the previous logging. Also, I need to check if `Terminal.IsConsolePresent` before add the `ConsoleLogger` in the static class `Logger`.

**Comment by geoperez** [comment]
There is another issue... Previous Console Logger in Terminal was writing directly to `System.Diagnostics.Debug` when Debugger was present. 

I have to add a proper `DebuggerLogger` class and register this target when the same logic.

**Comment by geoperez** [comment]
The nuget was released as version 2.1.0

**Comment by AbeniMatteo** [comment]
Thank you.
